src/renderers/shared/fiber/isomorphic/ReactPortal.js

```diff
@@ -27,6 +27,7 @@ exports.createPortal = function(
   implementation: any,
   key: ?string = null,
 ): ReactPortal {
+  containerInfo._reactPortalContainer = {};
```

Comment:
I also wonder if this means `createPortal(<div />, null)` is going to throw. Which I think ideally it should, but I’m not sure if this is the case now. If we currently silently skip it, it might break product code that accidentally relies on this. I would expect master to still throw in this case though, just later. But we need to verify. Ideally we’d add an early invariant in the future but that seems out of scope.